recipe/meta.yaml

```diff
@@ -1,33 +1,51 @@
 {% set name = "nbclassic" %}
-{% set version = "0.3.5" %}
+{% set version = "0.4.8" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 99444dd63103af23c788d9b5172992f12caf8c3098dd5a35c787f0df31490c29
+  sha256: c74d8a500f8e058d46b576a41e5bc640711e1032cf7541dde5f73ea49497e283
 
 build:
   number: 0
-  noarch: python
-  skip: True  # [s390x]
+  skip: True  # [py<37]
```

Comment:
is this nbconvert on the SOW for s390x?  Not that I am opposed to have it also for s390x, but we shouldn't not extend the list of maintained packages on this arch without need

Reply:
`notebook 6.5.2` requires `nbclassic >=0.4.7` https://github.com/AnacondaRecipes/notebook-feedstock/pull/29#issue-1445328726, and we maintain `notebook` on `s390x` platform